sdk/aqueduct/tests/decorator_local_method_test.py

```diff
@@ -0,0 +1,20 @@
+from aqueduct.decorator import op,metric,check
+from aqueduct.tests.utils import (
+    default_table_artifact,
+)
+inp = default_table_artifact()
+
+@op
+def op_fn_with_parentheses(df):
+    pass
+print(op_fn_with_parentheses.local.__name__)
+
+@check()
+def check_fn_with_parentheses(df):
+    pass
+print(check_fn_with_parentheses.local)
+
+@metric()
+def metric_fn_with_parentheses(df):
+    pass
+print(metric_fn_with_parentheses.local())
```

Comment:
these aren't really tests, they just print stuff but don't check the output. Since `.local()` is a user-facing feature, I'd prefer if we wrote some integration tests for them (see `integration_tests/sdk`). Lemme know if you need help figuring out how to run these. Check out the README here: https://github.com/aqueducthq/integration-test (I need to port this over to the OSS)